GameData/RP-0/Tree/EntryCostModifiers.cfg

```diff
@@ -31,16 +31,17 @@ ENTRYCOSTMODS
 	PumpReignition = 20000 // Ability to Relight
 	Gimbal = 0 // Given it's standard in turbopump engines, I think this can be applied only for PF ones
 
+	//Solids
 	SolidsHollow = 5000 // WW2 US tech
 	SolidsComposite = 5000 // PSPC - composite propellant, rubberized.
 	SolidsFiber = 20000 // Starting with Altair: fiberglass casing
 	SolidsAluminum = 50000, SolidsComposite, SolidsHollow // addition of Aluminum
 	SolidsTVC = 100000 // Gas injection, jet vanes, whatever.
-	SolidsPBAA = 15000, SolidsFiber, SolidsAluminum
-	SolidsPUPE = 5000, SolidsFiber, SolidsAluminum
-	SolidsCTPB = 10000, SolidsFiber, SolidsAluminum
-	SolidsPBAN = 50000, SolidsPBAA
-	SolidsHTPB = 50000, SolidsCTPB
+	SolidsPBAA = 15000, SolidsFiber, SolidsAluminum //Polybutadiene Acrylic Acid
+	SolidsPUPE = 5000, SolidsFiber, SolidsAluminum //???
```

Comment:
This is Aerojet's Polyurethane binder (polyurethane propylethyl something IIRC) as used on Polaris (and Algol).